src/web_scraper/web_scraper.py

```diff
@@ -0,0 +1,125 @@
+"""
+
+Description: this program will eventually be responsible for extracting RSS senate/house bill metadata and DOCX full bill, and then uploading to supabase table
+
+
+"""
+
+from docx import Document
+from supabase import create_client, Client
+from datetime import datetime
+import feedparser
+import os
+import re
+
+senate_rss_feed = "https://www.legis.state.pa.us/WU01/LI/RSS/SenateBills.xml"
+house_rss_feed = "https://www.legis.state.pa.us/WU01/LI/RSS/HouseBills.xml"
+
+sb_api_url = "https://vsumrxhpkzegrktbtcui.supabase.co" #os.environ["API_URL"] 
+sb_api_key = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InZzdW1yeGhwa3plZ3JrdGJ0Y3VpIiwicm9sZSI6ImFub24iLCJpYXQiOjE2ODQ3OTU2NzUsImV4cCI6MjAwMDM3MTY3NX0.9lafalZT9FJW1D8DAuIMrsRX0Gs6204nV8ETfGslrqI" #os.environ["API_KEY"] 
```

Comment:
I just added a Github secret called "SUPABASE_API_KEY". While this code is still fresh, let's change the code so that it uses that secret rather than having the API key hardcoded into the script.